### PR TITLE
Link 3 icons

### PR DESCRIPTION
**Pull request by flame-0**
## Description
https://github.com/prathameshmm02/Calculator-inator
https://www.celsoazevedo.com/files/android/google-camera/dev-burial/
https://play.google.com/store/apps/details?id=app.grapheneos.pdfviewer.play

## Type of change
:x: Bug fix (non-breaking change which fixes an issue)
:white_check_mark: Icon addition (non-breaking change that adds/modifies Lawnicons's icons)
:x: General change (non-breaking change that doesn't fit the above categories like copyediting)

### Icons linked
* Calculator-inator (linked `com.inator.calculator` to `@drawable/calculator`)
* Camera (linked `com.GoogleCamera.Burial` to `@drawable/camera`)
* PDF Viewer (linked `app.grapheneos.pdfviewer` to `@drawable/grapheneos_pdf`)

### CODE REVIEW

**Review by Goooler — CHANGES_REQUESTED** [review]
Suggest you that 029e690f0061aa011a78c6eda12ab7dc1edb171f should be sent to a separate PR.

**Comment by Goooler** [comment]
This PR should be merged after #816, right?

**Comment by flame-0** [comment]
> Suggest you that [029e690](https://github.com/LawnchairLauncher/lawnicons/commit/029e690f0061aa011a78c6eda12ab7dc1edb171f) should be sent to a separate PR.

Done.

> This PR should be merged after #816, right?

Yes, first: #816, second: #817, and third: #818.

**Comment by Goooler** [comment]
Please rebase the master and just keep changes in 1960a0a47d38720324a68c50d2dc4e1268a4ef47.

**Comment by Goooler** [comment]
This PR was closed by no changes push, please address 1960a0a47d38720324a68c50d2dc4e1268a4ef47 to a new one.

**Comment by flame-0** [comment]
Sorry, I am confused. What sort are you using?

**Comment by Goooler** [comment]
Sorted by AS, you just need to insert 3 changed lines.

File: app/src/main/res/xml/grayscale_icon_map.xml [hunk 1]
```diff
@@ -458,6 +460,7 @@
     <icon drawable="@drawable/grab" package="com.grabtaxi.passenger" name="Grab" />
     <icon drawable="@drawable/grapheneos_apps" package="app.grapheneos.apps" name="Apps" />
     <icon drawable="@drawable/grapheneos_auditor" package="app.attestation.auditor" name="Auditor" />
+    <icon drawable="@drawable/grapheneos_pdf" package="app.grapheneos.pdfviewer" name="PDF Viewer" />
```
Comment (flame-0) [inline app/src/main/res/xml/grayscale_icon_map.xml:463]: The line 463 is from official GitHub repository of GrapheneOS, and the line 464 is from Google Play. The original application name is PDF Viewer, no "Secure"
image

File: app/src/main/res/xml/grayscale_icon_map.xml [hunk 2]
```diff
@@ -458,6 +460,7 @@
...
     <icon drawable="@drawable/grapheneos_pdf" package="app.grapheneos.pdfviewer.play" name="Secure PDF Viewer" />
```
Comment (flame-0) [inline app/src/main/res/xml/grayscale_icon_map.xml:464]: This must be rename to `PDF Viewer` only.
image

Comment (Goooler) [reply app/src/main/res/xml/grayscale_icon_map.xml:464]: I just added what you added, if there is an issue, please send a new PR to fix.

Comment (Goooler) [reply app/src/main/res/xml/grayscale_icon_map.xml:464]: #820

**Pull request merged**